index.bs

```diff
@@ -29,14 +29,24 @@ Boilerplate: omit issues-index, omit conformance, omit feedback-header
 
 <pre class="link-defaults">
 spec: infra; type: dfn; text: list
+spec: url; type: interface; text: URL
+spec: dom; type: interface; text: Document
 </pre>
 
 <pre class="anchors">
 urlPrefix: https://html.spec.whatwg.org/; spec: HTML
     type: interface
-        text: Audio; url: dom-audio
-        text: Option; url: dom-option
-        text: Image; url: dom-image
+        text: HTMLAudioElement; url: htmlaudioelement
+        text: HTMLOptionElement; url: htmloptionelement
+        text: HTMLImageElement; url: htmlimageelement
+        text: onmouseenter; for: Document; url: handler-onmouseenter
+        text: onmouseleave; for: Document; url: handler-onmouseleave
+        text: onreadystatechange; for: Document; url: handler-onreadystatechange
+urlPrefix: https://fullscreen.spec.whatwg.org/; spec: FULLSCREEN
+    type: interface;
+        text: fullscreenEnabled; for: Document; url: dom-document-fullscreenenabled
+        text: fullscreen; for: Document; url: dom-document-fullscreen
+        text: fullscreenElement; for: DocumentOrShadowRoot; url: dom-document-fullscreenelement
```

Comment:
I think these are exported by default but that fullscreen isn't picked up by Shepherd? Maybe @plinss or @TabAtkins can confirm and fix if my assumption is correct.